tests/Behat/WebtoolsAnalyticsConfigContext.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools\Behat;
+
+use Drupal\DrupalExtension\Context\ConfigContext;
+
+/**
+ * Class DrupalContext.
+ */
+class WebtoolsAnalyticsConfigContext extends ConfigContext {
+
+  /**
+   * Configures the CAS module to use Drupal login.
```

Comment:
Not correct comment.
`Apply settings for the Webtools Analytics configuration.`